clamav/model.py

```diff
@@ -123,3 +125,45 @@ def age(self) -> int:
 
     def state(self) -> ClamAVHealthState:
         return ClamAVHealthState(self.raw['state'])
+
+
+class ClamAVScanEventTypes(enum.Enum):
+    ERROR = 'error'
+    RESULT = 'result'
+
+
+class ClamAVError(ModelBase):
+    def status_code(self) -> int:
+        '''Return the HTTP/1.1 status code that was given by ClamAV
+        '''
+        return self.raw['code']
+
+    def message(self) -> str:
+        return self.raw['message']
+
+
+class ClamAVScanEventClient(object):
```

Comment:
a brief hint as to what this client is for might help. naming nit: maybe better call it ClamAVScanEventSSEClient? not sure, though. gets lengthy..